### PR TITLE
Add How To for testing json path queries.

### PR DESCRIPTION
**Pull request by whitingjr**
This PR closes the task to add json path queries as a How To.

### CODE REVIEW

File: docs/howto/test-json-path.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# test JSON Path on Dataset How to
```
Comment (rvansa) [inline docs/howto/test-json-path.md:1]: This title wrong order of words seems to have.

File: docs/howto/test-json-path.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+ Datasets transformed from Runs are available to be used in Reports or change detection. Manipulating Run data that has relevant data into arrays or scalar value ready for these tasks. The ability to query JSON data and test the search paths is available using the Datasets view. The view provides immediate feedback on the success or otherwise of the query.
```
Comment (rvansa) [inline docs/howto/test-json-path.md:3]: Technically speaking it's Labels that you use in Reports or CD, Datasets are only intermediate representation. So I guess you might want to try certain JSON Path to use it in Label, you won't access Dataset directly anywhere.
The second sentence is missing a verb?

**Review by rvansa — COMMENTED** [review]
I am not sure if this really covers the intended topic; ability to execute JSON Paths in this view is there to let you explore the data. When you want to see if given JSON Path extracts the data you want you should rather go to the Label definition and press the 'Try it' button next to the extractor.

File: docs/howto/test-json-path.md (hunk 2 above)
Comment (shivam-sharma7) [reply docs/howto/test-json-path.md:3]: @rvansa close this

**Pull request closed without merging**